apps/aehttp/src/swagger/swagger_server.erl

```diff
@@ -1,7 +1,7 @@
 -module(swagger_server).
 
 
--define(DEFAULT_ACCEPTORS_POOLSIZE, 10).
+-define(DEFAULT_ACCEPTORS_POOLSIZE, 100).
```

Comment:
Unwanted change I guess?

Reply:
Default Swagger autogenerated poolsize. I could revert it but this is manually modifying an autogenerated file (and we would continue doing so in the future or push it upstream)

Reply:
I see change not present in current version - thanks.